tests/routers/test_auth.py

```
@@ -0,0 +1,179 @@
+    for i in range(5):
+        client.post("/api/v1/auth/login", json={"username": "a", "password": "b"})
+
+    response = client.post("/api/v1/auth/login", json={"username": "a", "password": "b"})
+    assert response.status_code == 429
+    assert "Too many requests" in response.json()["detail"]
+
+@pytest.fixture
+def authenticated_client(client: TestClient, db_session: Session):
+    """Fixture to get an authenticated client."""
+    username = "authtestuser"
+    password = "password"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "password"

(S105)